src/components/Icons/Twemoji/Mountain.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const Mountain = props => (
+  <svg viewBox="0 0 45 45" width={45} height={45} {...props}>
+    <g fill="none">
+      <path
+        fill="#292F33"
+        d="M24 44.346L15.406 14s-1.786.995-3.275 2.14c-.393.303-2.486 3.649-2.907 3.976-2.216 1.719-4.468 3.714-4.77 4.14C3.878 25.074 0 45 0 45l24-.654z"
+      />
+      <path
+        fill="#4B545D"
+        d="M40 44l-6.9-25.937L22.433 4l-6.741 5.507-1.219 1.539c-.51 1.025-2.078 7.017-2.078 7.017L10.828 24.3 3.93 37.504 1 43.726 40 44z"
+      />
+      <path
+        fill="#FFF"
+        d="M4 37s9.973-14.568 10-15.385c.026-.816-1.452-4.615-1.452-4.615l-1.583 6.417L4 37z"
+      />
+      <path
+        fill="#292F33"
+        d="M14.963 42h2.481l2.482-20.784 1.53-2.37 4.44-6.908.116-1.938-4.078 5.877-3.25 4.682-2.962 9.715-.76 2.489-2.221 4.135L10 42h4.963zm14.961-23.246l-2.438 7.782-1.357 4.907 3.464 10.062L34 41.423s-5.39-9.155-5.39-9.98c0-.158.776-3.545.776-3.545l1.357-5.98.349-2.021-1.168-1.143z"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 8. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)